src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,165 @@
+
+### Wallets
+
+User-wallets can be browser extensions, desktop and mobile apps or smart contracts on Ethereum. There is active research into social recovery wallets that reduce some of the risk associated with individual-user key management.
+
+### Links to current research
+
+[Three Attacks on PoS Ethereum](https://arxiv.org/abs/2110.10086)
+
+[ethresear.ch Security](https://ethresear.ch/tag/security)
+
+[Ethresear.ch ZK](https://ethresear.ch/c/zk-s-nt-arks/13)
+
+[zkp.science](https://zkp.science/)
+
+[Zero Knowledge podcast](https://zeroknowledge.fm/)
+
+## Community, Education and Outreach
+
+Onboarding new users onto Ethereum requires new educational resources and approaches to outreach. This might include blog posts and articles, books, podcasts, memes, teaching resources events and anything else that builds communities, welcomes new starters and educates people about Ethereum.
+
+### UX/UI
+
+For Ethereum to onboard more people the UX/UI must be improved. This requires designers and UI and product experts to re-examine the design of wallets and apps.
+
+### Links to current research
+
+[Ethresear.ch UX/UI](https://ethresear.ch/c/ui-ux/24)
+[Ethereum.org community page](https://ethereum.org/en/learn/)
+
+## Economics
+
+There are complex crypto-economic factors relating to Ethereum's native asset (ether) and the tokens built on top of it (for example NFTs and ERC20 tokens).
+
+### PoS incentives
+
+When Ethereum transitions to proof of stake, its native asset (ether) will be used as collateral by validators. The cryptoeconomics of this
+
+### Liquid staking and derivatives
+
+Liquid staking allows users with less than 32 ETH to receive staking yields by swapping ether for a token representing staked ether that can be used in DeFi. However, the incentives and market dynamics associated with liquid staking as well as its effect on Ethereum's security (e.g. centralization risks, )
+
+### DeFi
+
+Decentralized finance (DeFi) is one of the primary classes of application built on top of Ethereum. DeFi aims to create composable "money legos" that allow users to store, transfer, lend, borrow and and invest crypto-assets using smart contracts. This is a fast-moving space that is constantly updating. Research into secure, efficient and accessible protocols is constantly needed.
+
+### Taxation and accounting
+
+Crypto taxation is a complicated issue that likely requires a software solution. There are some nascent crypto accounting project but this area requires much more attention.
+
+### Payment systems
+
+Using Ethereum for everyday purchases is not common right now. This is because of accessibility and cost issues. These are probably resolvable with additional R&D especially on top of Layer 2 infrastructure.
+
+### Links to current research
+
+[Ethresear.ch liquid staking](https://ethresear.ch/search?q=liquid%20staking)
+
+[Ethresear.ch Economics](https://ethresear.ch/c/economics/16)
+
+[Rotki crypto accounting](https://rotki.com/)
```

Comment:
I'd be hesitant to list specific products here... might lead to the type of link-building we're not looking for on this resource.